packages/ml-sdk/src/generated/models/missing-data.ts

```diff
@@ -84,8 +74,8 @@ export interface MissingAttributes {
   readonly attributeCoverage?: AttributeCoverage
 }
 export interface MissingAttributesMeta {
-  readonly productLevel: MissingAttributesProductLevel
-  readonly variantLevel: MissingAttributesVariantLevel
+  readonly productLevel: MissingAttributesDetails
+  readonly variantLevel: MissingAttributesDetails
```

Comment:
why is this renamed? is it just internal? 🤔

Reply:
actually here the types are the same, they just have different descriptions, this is why i decided to make the same type for both.